tests/integration/static_ip_address_test.py

```diff
@@ -25,9 +25,11 @@
 from .testlib import statelib
 from .testlib.statelib import INTERFACES
 
-
+# TEST-NET addresses: https://tools.ietf.org/html/rfc5737#section-3
 IPV4_ADDRESS1 = '192.0.2.251'
 IPV4_ADDRESS2 = '192.0.2.252'
+IPV4_ADDRESS3 = '198.51.100.249'
```

Comment:
I do not understand why additional addresses were needed, using the existing ones could be enough.
But it is not harming anything, just raising a note for the future.

Reply:
Maybe only one extra address was needed. My motivation was to avoid problems with setting the same address to different interfaces (eth1 and eth2). However, this should probably also be a test case - I wonder how nmstate will handle this.

Reply:
Yes, this is a good test idea. @SvetaHBar could you please check this?